### DISCUSSION TITLE
Another shader options question

### DISCUSSION THREAD

**Discussion by sociofall** [IrisShaders/Iris#471] [Q&A] [answered]
Sorry for dummy question, but idk how to enable motion blur on Complementary Shaders. Tried editing shaders.properties profiles and adding it to #Force Video Settings tab but no way. Help me pleease :)

**Comment by WereWolfix** [comment, marked as answer]
I have found a way to change the settings from the shaders files. Obviously you must have unzipped the zip file. I give an example with BSL_shader. Go inside the BSL / shaders / lib folder, inside you will find a file called setting.glsl, open it with a text editor program. All the settings preceded by // will be disabled by default so if you want to have them active you have to delete the // same thing vice versa if you want to disable them add the // in front of the setting
